app/scripts/modules/google/src/help/gce.help.ts

```diff
@@ -195,6 +195,17 @@ const helpContents: { [key: string]: string } = {
     '<p>(Optional) The name of a Google account configured within Rosco. If left blank, the first configured account will be used.</p>',
   'pipeline.config.gce.bake.baseImage':
     '<p>(Optional) A GCE image name. For example: ubuntu-1204-precise-v20150910.</p>',
+  'gce.loadBalancertype.Network': `
```

Comment:
```suggestion
  'gce.loadBalancerType.Network': `
```
Let's be consistent with the camel casing and capitalize "Type"